tests/glsl/ssboMemberMemoryQualifier.slang

```diff
@@ -0,0 +1,48 @@
+//TEST:SIMPLE(filecheck=CHECK-GLSL): -stage compute -entry computeMain -allow-glsl -target glsl
+//TEST:SIMPLE(filecheck=CHECK-SPV): -stage compute -entry computeMain -allow-glsl -target spirv -emit-spirv-directly
+
+volatile buffer Block1
+{
+    restrict uint data1;
+    readonly uint data2;
+    readonly writeonly uint data3;
```

Comment:
Can we test about "readonly writeonly" keyword with ImageSize and .length() as the OpenGL spec says?
```
A variable could be qualified as both readonly and writeonly, disallowing both read and write.
Such variables can still be used with some queries, for example imageSize() and .length()
```

Reply:
.length is not implemented and not currently trivially implementable (due to swizzle implementation). imageSize is implemented & tested with `readonly writeonly`.